File: backend/Models/ReservationTimesheet.cs
namespace backend.Models
{
    public class ReservationTimesheet
    {
        public int TimesheetId { get; set; }
        public DateOnly Date {  get; set; }
        public TimeOnly StartTime { get; set; }
        public TimeOnly EndTime { get; set; }
        public bool? IsTournament { get; set; }

        public int ObjectId { get; set; }
        public ObjectType ObjectType { get; set; }
    }
}

File: frontend/src/shared/types/models/reservationTimesheet.ts
import { ObjectTypeDto } from "./objectType";

export interface CreateReservationTimesheetData {
    date: string,
    startTime: string,
    endTime: string,
    objectId: number | null,
}

export interface ReservationTimesheetDto {
    timesheetId: number,
    date: string,
    startTime: string,
    endTime: string,
    isTournament: boolean,
    objectType: ObjectTypeDto,
}